packages/maskbook/src/extension/options-page/DashboardComponents/CollectibleList/index.tsx

```diff
@@ -1,11 +1,13 @@
-import { Box, Button, makeStyles, Skeleton, Typography } from '@material-ui/core'
+import { Box, Button, CircularProgress, makeStyles, Skeleton, Typography } from '@material-ui/core'
 import { CollectibleCard } from './CollectibleCard'
 import { createERC1155Token, createERC721Token } from '../../../../web3/helpers'
 import type { WalletRecord } from '../../../../plugins/Wallet/database/types'
 import { useChainId } from '../../../../web3/hooks/useChainState'
 import { formatEthereumAddress } from '../../../../plugins/Wallet/formatter'
 import { createContext } from 'react'
 import type { Collectible } from '../../../../plugins/Wallet/types'
+import AutoResize from 'react-virtualized-auto-sizer'
+import { FixedSizeGrid } from 'react-window'
```

Comment:
Why you need `react-window` if you're using paginator?

Reply:
Due to part of the api interface, we can not correctly get the total of the data source, Paginator component can't correctly display the previous and next some state, so here we changed to use FixedList to achieve the effect of drop-down paging